File: examples/petstore/oseg_generated/php/DeleteUser_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    (new OpenAPI\Client\Api\UserApi($config))->deleteUser(
        username: null,
    );
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling User#deleteUser: '
        . print_r($e->getResponseObject());
}

File: examples/petstore/oseg_generated/csharp/DeleteUserDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class DeleteUserDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            new UserApi(config).DeleteUser(
                username: "my_username"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling User#DeleteUser: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
